backend/src/payloads/AppUser.ts

```diff
@@ -261,3 +261,49 @@ export class AppUserInterviewAvailability {
   @IsDateString()
   end: string;
 }
+
+export class InducteePostRequest {
+  @IsEmail()
+  readonly email: string;
+
+  @IsString()
+  readonly firstName: string;
+
+  @IsString()
+  readonly lastName: string;
+
+  @IsString()
+  readonly major: string;
+
+  @IsString()
+  readonly graduationYear: string;
+
+  @IsString()
+  readonly password: string;
+
+  @IsString()
+  readonly preferredName: string;
```

Comment:
`preferredName`, `pronoun` and `customPronoun` are optional on the new inductee account sign up page so they should be optional here as well. While you're at it, do the same thing here for AppUserSignupRequest

Reply:
Actually I take it back, don't touch AppUserSignupRequest. You can just change the payload class you made